esphome/components/emporia_vue/sensor.py

```diff
@@ -11,16 +11,16 @@
     CONF_VOLTAGE,
     DEVICE_CLASS_ENERGY,
     DEVICE_CLASS_VOLTAGE,
-    ICON_EMPTY,
     STATE_CLASS_MEASUREMENT,
     UNIT_WATT,
     UNIT_VOLT,
 )
 
 # TODO: Remove this - It's only added so we can test the component using the External_component configuration
-CONF_CT = "ct"
+CONF_CT_CLAMPS = "ct_clamps"
```

Comment:
I think a plural word matches `phases` better; and I think `ct_clamps` is more clear than `cts` or `ct`

Reply:
I agree 💯